scanner-function.tf

```diff
@@ -1,7 +1,7 @@
 resource "aws_lambda_function" "antivirus-scanner" {
   function_name = "${var.env}-bucket-antivirus-scanner"
   timeout       = 300
-  memory_size   = 1024
+  memory_size   = 3008
```

Comment:
`3072` would be more base_2 technically

Reply:
Selected 3008 because it is the current maximum in lambda. https://docs.aws.amazon.com/lambda/latest/dg/gettingstarted-limits.html

Reply:
That is a direct attack to all base2 enthusiasts